routers/user/home.go

```diff
@@ -194,7 +196,21 @@ func Issues(ctx *context.Context) {
 		page = 1
 	}
 
-	repoID := ctx.QueryInt64("repo")
+	// Regexp: Match all square brackets, and plus signs in order to handle pagination
+	re := regexp.MustCompile(`[\[\]\+]+`)
```

Comment:
Should this be compiled each time?

Reply:
@techknowlogick is there a special place where RegExp are stored?
if not I'll move it out of this func but leave it in home.go

Reply:
> 
> 
> @techknowlogick is there a special place where RegExp are stored?
> if not I'll move it out of this func but leave it in home.go

Check this example:

https://github.com/go-gitea/gitea/blob/8f09aff427add98cbad0254c4849ea6eee34e070/models/issue_label.go#L20

Reply:
done